src/zoom/zoomRulesLowMat.ts

```diff
@@ -34,6 +34,31 @@ export default (function zoom_rules_low_mat(
     newZoomData.viz_offcenter =
       (viz_dim.canvas[canvas_dim] * viz_dim.offcenter[axis]) / 2;
   }
+
+  // remove gap when zooming in the clustergrammer in y-axis
+  const y_axis_low_offset = 13; // offset at the bottom
+  const y_axis_high_offset = 11.5; // offset at the top
+
```

Comment:
Is this from the main clustergrammer or is this our code? Where did we get these numbers from?

Reply:
Yes its our code I have tried with different numbers for offset and came to these numbers

Reply:
We've discussed that random eyeballed numbers are not a true solution. Wouldn't this affect the weird offset we see with tooltip?

Reply:
tooltip was already misaligned and I was not able to find other way to reduce the gap.

Reply:
What I mean is - would these numbers potentially throw off the tooltip even more? If there was no other way, we could have discussed this. Random numbers that don't have a true rationale behind them are not a fix. Do you know for sure that this was the exact offset value? Do we know what was causing the offset? The numbers are too random and this is just delaying having to find a solution for later

Reply:
It can affect the tooltip too. Yes they are the exact offset value but i cannot figure out what was causing the issue.